client-web/test/exlibris-app/exlibris-app_store_test.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport"
+          content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>exlibris-app store test</title>
+
+    <script src="./test_variables.js"></script>
+    <script src="../../bower_components/webcomponentsjs/webcomponents-loader.js"></script>
+    <script src="../../bower_components/web-component-tester/browser.js"></script>
+    <script src="../../node_modules/redux/dist/redux.js"></script>
+
+    <link rel="import" href="../../src/exlibris-app/exlibris-app.html">
+</head>
+<body>
+
+<test-fixture id="BasicTestFixture">
+    <template>
+        <exlibris-app></exlibris-app>
+    </template>
+</test-fixture>
+
+<script>
+    suite('exlibris-app', function () {
+        let mock;
+
+        setup(() => {
+            mock = sinon.mock(ExlibrisApp.store);
+        });
+
+        teardown(() => {
+            mock.restore();
+        });
+
+        test('should set allBooks value from state', function () {
+            mock.expects("getState").returns({allBooks: [...ExlibrisAppTest.BOOKS]});
+            const element = fixture('BasicTestFixture');
+            assert.deepEqual(element.allBooks, ExlibrisAppTest.BOOKS);
+            mock.verify();
+        });
+        test('should set borrowedBooks value from state', function () {
+            mock.expects("getState").returns({borrowedBooks: [...ExlibrisAppTest.BOOKS]});
+            const element = fixture('BasicTestFixture');
+            assert.deepEqual(element.borrowedBooks, ExlibrisAppTest.BOOKS);
+            mock.verify();
+        });
+        test('should set reservedBooks value from state', function () {
+            mock.expects("getState").returns({reservedBooks: [...ExlibrisAppTest.BOOKS]});
+            const element = fixture('BasicTestFixture');
+            assert.deepEqual(element.reservedBooks, ExlibrisAppTest.BOOKS);
+            mock.verify();
+        });
+        test('should set shouldReturnBooks value from state', function () {
+            mock.expects("getState").returns({shouldReturnBooks: [...ExlibrisAppTest.BOOKS]});
+            const element = fixture('BasicTestFixture');
+            assert.deepEqual(element.shouldReturnBooks, ExlibrisAppTest.BOOKS);
+            mock.verify();
+        });
+        test('should set overdueBooks value from state', function () {
+            mock.expects("getState").returns({overdueBooks: [...ExlibrisAppTest.BOOKS]});
+            const element = fixture('BasicTestFixture');
+            assert.deepEqual(element.overdueBooks, ExlibrisAppTest.BOOKS);
+            mock.verify();
+        });
+        test('should set expectedSoonBooks value from state', function () {
+            mock.expects("getState").returns({expectedSoonBooks: [...ExlibrisAppTest.BOOKS]});
+            const element = fixture('BasicTestFixture');
+            assert.deepEqual(element.expectedSoonBooks, ExlibrisAppTest.BOOKS);
```

Comment:
Well, these tests are testing the functionality of ReduxMixin which is from a separate library and is tested in it.